crowbar_framework/config/initializers/progress_logger.rb

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2017, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Logger
+  def progress(progname, &block)
+    SEV_LABEL << "PROGRESS"
+    idx = SEV_LABEL.size - 1
+
+    log_file = if Rails.env.production?
+      File.join(ENV["CROWBAR_LOG_DIR"], "progress.log")
+    else
+      Rails.root.join("log", "progress.log")
+    end
+
+    logger = Logger.new(log_file)
+    logger.formatter = ::Crowbar::Logger::Formatter.new
+    logger.info(progname)
```

Comment:
Is this part necessary? Seems like we are logging twice no? Other logger levels do nothing like this so could you clarify why is this needed?